index.html

```diff
@@ -40,12 +40,169 @@
     </style>
   </head>
   <body>
-    <!--
+    <header>
+      <h2>Travel Blog</h2>
+      <p>Site Name: Travel Blog</p>
+
+      <img 
+      src="https://img.freepik.com/premium-vector/travel-blog-logo-using-pen-plane-as-negative-space_560919-194.jpg"
+      alt="Travel Blog Logo"
+      />
+
+      
+      <nav>
+      <h2>Menu</h2>
+        <ul>
+          <li><a href="#">Home</a></li>
```

Comment:
I'm going to challenge you on this `h2` 

Since it's still in the header, I think this is more appropriately marked up as an `h3`